pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -252,7 +258,7 @@ private void receiveMessageFromConsumer(ConsumerImpl<T> consumer, boolean batchR
             }
             // Process the message, add to the queue and trigger listener or async callback
             messages.forEach(msg -> {
-                if (isValidConsumerEpoch((MessageImpl<T>) msg)) {
+                if (isValidConsumerEpoch((MessageImpl<T>) msg) && !duringSeek) {
```

Comment:
We'd better add a DEBUG level log here to know if the messages were skipped during the seek operation. It will help us troubleshoot some problems related to the message that was not received from the user's perspective.

Reply:
I think INFO log would be better. It's a rare case to come here because the check is already done at the beginning of `receiveMessageFromConsumer`. Even if it came here, the maximum size of `messages` is 1 or `batchMaxMessages` (by default 1000).